ql/src/test/queries/clientpositive/structin.q

```diff
@@ -21,3 +21,14 @@ IN (
 struct('1234-1111-0074578664','3'),
 struct('1234-1111-0074578695',1)
 );
+
+CREATE TABLE test_struct
+(
+  f1 string,
+  demo_struct struct<f1:string, f2:string, f3:string>,
+  datestr string
+);
+
```

Comment:
Can we insert a few rows here and add the SELECT query? Just in case that `IS NOT NULL` predicate in the explain goes away with a future change, we do not confuse it with an optimization.

Reply:
I am unable to add a `null` value into struct column. `cast as <struct>` doesn't work. I have added a test to insert someone data.